client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/ILMDocumentationIT.java

```diff
@@ -226,6 +230,82 @@ public void onFailure(Exception e) {
         assertTrue(latch.await(30L, TimeUnit.SECONDS));
     }
 
+    public void testRetryPolicy() throws Exception {
+        RestHighLevelClient client = highLevelClient();
+
+        // setup policy to immediately fail on index
+        {
+            Map<String, Phase> phases = new HashMap<>();
+            Map<String, LifecycleAction> warmActions = new HashMap<>();
+            warmActions.put(ShrinkAction.NAME, new ShrinkAction(1));
+            phases.put("warm", new Phase("warm", TimeValue.ZERO, warmActions));
+
+            Map<String, LifecycleAction> deleteActions =
+                Collections.singletonMap(DeleteAction.NAME, new DeleteAction());
+            phases.put("delete", new Phase("delete",
```

Comment:
What's the reasoning behind having a delete phase here?

Reply:
true, no need. didn't think it hurts. but it is also unnecessary LOC